style.css

```diff
@@ -0,0 +1,252 @@
+body {
+    display: flex;
+    position: relative;
+    width: 1440px;
+    height: 1000px;
+    background-color: #141b1c;
+}
+.section_1 {
+    position: absolute;
+    left: 132px;
+    right: 928px;
+    top: 115px;
+    bottom: 115px;
+    width: 380px;
+    height: 770px;
+    background-color: #334247;
+    box-shadow: 0px 6px 6px rgba(0, 0, 0, 0.087466);
+}
+.section_2 {
+    position: absolute;
+    left: 530px;
+    right: 530px;
+    top: 115px;
+    bottom: 115px;
+    background-color: #334247;
+    width: 380px;
+    height: 770px;
+    overflow: hidden;
+}
+.Popular-background {
+    position: absolute;
+    width: 25.01px;
+    height: 147.08px;
+    background-color: #dc3f45;
+    position: absolute;
+    top: -38.41px;
```

Comment:
do you really need negative positioning?

Reply:
use flexbox